src/anyp/Uri.h

```diff
@@ -104,6 +104,9 @@ class Uri
     /// the static '*' pseudo-URI
     static const SBuf &Asterisk();
 
+    /// %-encode characters in a buffer
+    static SBuf Encoder(const SBuf &, const CharacterSet &);
```

Comment:
The method encodes the given string. It does not create an encoder object -- it is not a constructor. I suggest naming it using a verb, based on the action it performs:

```suggestion
    static SBuf Encode(const SBuf &, const CharacterSet &);
```

Reply:
Done in  3f2b82cdd42f6c8a2e9ae1e74ae2c763c57afc9f

Reply:
> Done in 3f2b82c

That commit missed the method declaration quoted in the change request.

Reply:
Oops, fixed in 5fd522570c344b33a7b4b0af9e37067deb7e2d24.